### PR TITLE
[Term Entry] Swift Arrays .allSatisfy()

### PR DESCRIPTION
**Pull request by d-fontanez**
### Description

- Added the allSatisfy() method for Swift Array terms
- Closes issue #2449 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  - 'Mobile Development'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+  - 'SwiftUI'
+CatalogContent:
+  - 'learn-Swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+** .allSatisfy() ** is a Swift method that checks if all the elements in an array satisfies a given condition.
```
Comment (SSwiniarski) [inline content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md:15]:
```suggestion

**`.allSatisfy()`** is a Swift method that checks if all the elements in an array satisfies a given condition. 
```

File: content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  - 'SwiftUI'
+CatalogContent:
+  - 'learn-Swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+** .allSatisfy() ** is a Swift method that checks if all the elements in an array satisfies a given condition. 
+
+## Syntax
+```pseudo
+array.allSatisfy(condition)
```
Comment (SSwiniarski) [inline content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md:19]: We need a sentence here explaining what `condition` is, and what can be put there.

File: content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+```
+
+## Example
```
Comment (SSwiniarski) [inline content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md:22]:
```suggestion
## Example

```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@d-fontanez I did a first review and left a few comments.

File: content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md (hunk 2 above)
Comment (d-fontanez) [reply content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md:21]: Sentence added, please let me know if changes are needed

File: content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+```swift
```
[the remark this answers is not in the document]
Comment (d-fontanez) [reply content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md:28]: Added, please let me know if changes are needed

**Review by SSwiniarski — APPROVED** [review]
@d-fontanez I did a couple of edits to fix your code blocks. Now this is ready for a second review.

File: content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
+---
+Title: '.allSatisfy()'
+Description: 'Returns a Boolean expression indicating whether every element of a sequence satisfies a given condition.'
+Subjects:
+  - 'IOS'
```
Comment (caupolicandiaz) [inline content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md:5]:
```suggestion
  - 'iOS'
```

File: content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  - 'Mobile Development'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+  - 'SwiftUI'
+CatalogContent:
+  - 'learn-Swift'
```
Comment (caupolicandiaz) [inline content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md:12]:
```suggestion
  - 'learn-swift'
```

File: content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+**`.allSatisfy()`** is a Swift method that checks if all the elements in an array satisfies a given condition.
```
Comment (caupolicandiaz) [inline content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md:16]:
```suggestion
**`.allSatisfy()`** is a Swift method that checks if all the elements in an array satisfy a given condition. 
```

File: content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+**`.allSatisfy()`** is a Swift method that checks if all the elements in an array satisfies a given condition. 
+
+## Syntax
+
+```pseudo
+array.allSatisfy(condition)
+```
+
+`condition` runs a true/false test on an array or set of given input values.
```
Comment (caupolicandiaz) [inline content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md:24]:
```suggestion
`condition`: An expression that is used to test the values in the array.
```

File: content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+
+## Example
+
+In the example below, an array was created called numbers with the values of 2, 4, 6, 8. We want to know if all the values in our array is even so we use the .allSatisfy method to check each value against a condition of our choosing. The condition in this example confirms if a value is even by verifying it is divisble by 2. If all values in the array satsify this condition the output will be true. 
+
+```swift
+//create array
```
Comment (caupolicandiaz) [inline content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md:31]:
```suggestion
// Create an array
```

File: content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+let numbers = [2, 4, 6, 8]
+//create condition
```
Comment (caupolicandiaz) [inline content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md:33]:
```suggestion
// Specify the condition to be tested
```

File: content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+
+```pseudo
+array.allSatisfy(condition)
+```
+
+`condition` runs a true/false test on an array or set of given input values.
+
+## Example
+
+In the example below, an array was created called numbers with the values of 2, 4, 6, 8. We want to know if all the values in our array is even so we use the .allSatisfy method to check each value against a condition of our choosing. The condition in this example confirms if a value is even by verifying it is divisble by 2. If all values in the array satsify this condition the output will be true.
```
Comment (caupolicandiaz) [inline content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md:28]:
```suggestion
In the example below, an array called `numbers` is declared. Then the `.allSatisfy()` method  is used to evaluate if the elements of `numbers` are all even (divisible by two).
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@d-fontanez I've completed my initial review. Please update the entry, and tag me when you've finished making the changes. Thanks

**Comment by d-fontanez** [comment]
Thank you @caupolicandiaz . The edits are done, please let me know if additional changes are needed.

**Review by caupolicandiaz — APPROVED** [review]
@d-fontanez @SSwiniarski Looks good to merge.

**Comment by SSwiniarski** [comment]
@d-fontanez Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/swift/arrays/allSatisfy

image

You're currently anonymous. If you don't wish to be anonymous, check out this forum thread on how to link your GitHub and Codecademy profiles: https://discuss.codecademy.com/t/my-contribution-is-live-but-im-not-being-shown-as-a-contributor-why-is-this/758036  **Note:** it may take a few minutes for the change to register on the entry page.

**Comment by d-fontanez** [comment]
@caupolicandiaz @SSwiniarski Thank you!

**Pull request merged**